plugins/cost-insights/src/components/CostOverviewCard/CostOverviewBreakdownChart.tsx

```diff
@@ -168,28 +167,26 @@ export const CostOverviewBreakdownChart = ({
           fill={color}
           onClick={() => setExpanded(true)}
           style={{
-            cursor: breakdown === 'Other' && !isExpanded ? 'pointer' : null,
+            cursor:
+              breakdown === 'Other' && !isExpanded ? 'pointer' : undefined,
           }}
         />
       );
     });
   };
 
-  const tooltipRenderer: ContentRenderer<TooltipProps> = ({
-    label,
-    payload = [],
-  }) => {
+  const tooltipRenderer: TooltipRenderer = ({ label, payload = [] }) => {
     if (isInvalid({ label, payload })) return null;
 
     const date =
       typeof label === 'number'
         ? DateTime.fromMillis(label)
         : DateTime.fromISO(label!);
     const dateTitle = date.toUTC().toFormat(DEFAULT_DATE_FORMAT);
-    const items = payload.map(p => ({
+    const items = payload.map((p, i) => ({
       label: p.dataKey as string,
-      value: formatGraphValue(p.value as number),
-      fill: p.fill!,
+      value: formatGraphValue(p.value as unknown as number, i),
```

Comment:
this cast looks scary :D could possibly have been `formatGraphValue(Number(p.value), i)`